src/utils.js

```diff
@@ -75,7 +75,22 @@ function getNetworkedEntity(entity) {
 module.exports.getNetworkedEntity = getNetworkedEntity;
 
 module.exports.takeOwnership = function(entity) {
-  getNetworkedEntity(entity).setAttribute("networked", { owner: NAF.clientId });
+  const networkedEntity = getNetworkedEntity(entity);
+  if(!networkedEntity) //bug? sometimes this is null
```

Comment:
`getNetworkedEntity` will return null when it has no ancestor with a `networked` component. It's good to check for this, but I don't think we need the comment.